HISTORY.md

```diff
@@ -1,27 +0,0 @@
-## 1.1.0 (2017-10-17)
```

Comment:
Why is this file being removed? If you think the changelog.md looks better without it, could you include the updated version in this PR?

Reply:
Removing a redundant file as the CHANGELOG.md already fills this role AND has the same content for these versions in there.

Reply:
CHANGELOG section for reference: https://github.com/voxpupuli/puppet-rsyslog/blob/master/CHANGELOG.md#110-2017-10-17

Reply:
It has the same contents *because* of HISTORY.md  It's a feature of github changelog generator.

Reply:
See https://github.com/github-changelog-generator/github-changelog-generator#migrating-from-a-manual-changelog